src/OpenSource.js

```diff
@@ -58,7 +59,7 @@ export default function OpenSource() {
             Design Systems team
           </LinkDark>
           <Text>.</Text>
-          <Text as="p">Copyright GitHub 2018.</Text>
+          <Text as="p">Copyright GitHub {CopyrightYear}.</Text>
```

Comment:
Since it doesn't take that much code to get the current year, we could probably fit this all in one line:

```suggestion
          <Text as="p">Copyright GitHub {new Date().getFullYear()}.</Text>
```

Reply:
Thank you!!! As a JS infant I always forget what can be accomplished inline in React in the correct syntax 🙈💪